src/pages/Editor.tsx

```diff
@@ -80,60 +74,33 @@ const Editor: React.FC = () => {
-                    <DialogHeader>
-                      <DialogTitle>Edit profile</DialogTitle>
-                      <DialogDescription>
-                        Make changes to your profile here. Click save when
-                        you're done.
-                      </DialogDescription>
-                    </DialogHeader>
-                    <div className="grid gap-4 py-4">
-                      <div className="grid grid-cols-4 items-center gap-4">
-                        <Label htmlFor="name" className="text-right">
-                          Name
-                        </Label>
-                        <Input
-                          id="name"
-                          value="Pedro Duarte"
-                          className="col-span-3"
-                        />
-                      </div>
-                      <div className="grid grid-cols-4 items-center gap-4">
-                        <Label htmlFor="username" className="text-right">
-                          Username
-                        </Label>
-                        <Input
-                          id="username"
-                          value="@peduarte"
-                          className="col-span-3"
-                        />
-                      </div>
-                    </div>
-                    <DialogFooter>
-                      <Button type="submit">Save changes</Button>
-                    </DialogFooter>
-                  </DialogContent>
-                </Dialog>
-                <DropdownMenuSeparator />
-                <DropdownMenuItem>Education</DropdownMenuItem>
-                <DropdownMenuItem>Work Experience</DropdownMenuItem>
-                <DropdownMenuItem>Extracurricular</DropdownMenuItem>
-                <DropdownMenuItem>Technical Skills</DropdownMenuItem>
-              </DropdownMenuContent>
-            </DropdownMenu>
+            <div className="flex items-center justify-between">
+              <DropdownMenu>
+                <DropdownMenuTrigger>
+                  <Button className="mt-1 ml-1" variant="outline">
+                    Add Resume Item
+                  </Button>
+                </DropdownMenuTrigger>
+                <DropdownMenuContent>
+                  <DropdownMenuLabel className="text-center">
+                    Item Type
+                  </DropdownMenuLabel>
+                  <DropdownMenuSeparator />
+                  <EducationItem></EducationItem>
+                  <DropdownMenuSeparator />
+                  <ExperienceItem></ExperienceItem>
+                  <DropdownMenuSeparator />
+                  <ExtracurricularItem></ExtracurricularItem>
+                  <DropdownMenuSeparator />
+                  <ProjectItem></ProjectItem>
+                </DropdownMenuContent>
+              </DropdownMenu>
+              <img
+                src={ExportImage}
+                alt="export"
+                className="mr-4 h-[30px] w-[30px]"
+              ></img>
```

Comment:
Is this going to become the generate pdf button? If so, we may want to have it somewhere on the right half of the screen so it's clear it corresponds to the resume they've built (this might be nitpicky)

Reply:
Good idea, I'll probably remove it for now and we can add it in again when we get a clearer picture of what the page will look like. (I'll keep the image file so we can use it later though).